src/test/java/com/wikia/webdriver/common/core/url/FandomUrlBuilder.java

```diff
@@ -1,35 +1,46 @@
 package com.wikia.webdriver.common.core.url;
 
 import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.core.configuration.EnvType;
 
 public class FandomUrlBuilder extends UrlBuilder {
 
-  private static final String FANDOM_URL = "fandom.wikia.com";
+  private static final String FANDOM_HOSTNAME = "fandom.wikia.com";
   private static final String ARTICLE_PATH = "articles";
+  private static final String TOPICS_PATH = "topics";
   public static final String ENV_PROD = "prod";
+  public final EnvType envType;
 
   private String browser;
-  private String env;
 
   public FandomUrlBuilder() {
     super();
-    this.env = Configuration.getEnv();
     this.browser = Configuration.getBrowser();
+    envType = Configuration.getEnvType(this.env); //Why not to pass EnvType in UrlBuilder constructor
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ATrailingCommentCheck)